app/src/main/res/values-nl/strings.xml

```diff
@@ -97,6 +97,13 @@
     <string name="contact_set_name">Geef contact een naam</string>
     <string name="contact_exists_message">Contact bestaat al, geef aub een unieke naam</string>
 
+    <!-- NFC -->
+    <string name="descr_gif">Dit zou een gif moeten zijn die nfc connection uitbeeldt.</string>
```

Comment:
`<string name="nfc_image_description">Image visualizing how to use NFC</string>`

FYI: the `android:contentDescription` is intended for people that are blind or have bad eyesight 😉